app/src/main/res/layout/activity_location_permission.xml

```diff
@@ -8,23 +8,58 @@
 
     <TextView
         android:id="@+id/textView2"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="232dp"
+        android:layout_width="315dp"
+        android:layout_height="61dp"
+        android:layout_marginTop="488dp"
+        android:layout_marginBottom="182dp"
         android:text="@string/require_locaiton_permission"
+        android:textAlignment="center"
+        android:textColor="#000000"
+        android:textSize="16sp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.533"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <ImageView
```

Comment:
Maybe add a contentDescription for the image ?

Reply:
You're right. Added it now.